pyproject.toml

```diff
@@ -101,6 +101,22 @@ filterwarnings = ["ignore::DeprecationWarning"]
 minversion = "6.0"
 markers = ["integration", "callbacks"]
 
+######################################
+# DOCS
+######################################
+
+[tool.hatch.envs.docs]
+dependencies = [
+    "mkdocs",
+    "mike",
+    "pymdown-extensions",
+]
+
+[tool.hatch.envs.docs.scripts]
+build = "mike deploy --push dev"
```

Comment:
We just realised that this is publishing to GH pages 🙈 
We'll fix this as part of #279